backend/spec/models/comment_spec.rb

```diff
@@ -52,6 +52,27 @@
         expect(comment.errors.full_messages).to eq ['内容を入力してください']
       end
     end
+
+    context 'contentが1025文字以上の場合' do
+      before { comment.content = 'a' * 1025 }
+
+      it 'valid?メソッドがfalseを返すこと' do
+        expect(comment).not_to be_valid
+      end
+
+      it 'errorsに「内容は1024文字以内で入力してください」と格納されること' do
+        comment.valid?
+        expect(comment.errors.full_messages).to eq ['内容は1024文字以内で入力してください']
```

Comment:
Idiomatic

以下のように記述した方がスマートかなと思いました〜！
```suggestion
        aggregate_failures do
          expect(comment).to be_invalid
          expect(comment.errors.full_messages).to eq ['内容は1024文字以内で入力してください']
        end
```

Reply:
aggregate_failuresを使用した記法に変更しました。
それに合わせて他のcommntのモデルのテストも同じ記法に統一しました！